app/models/search.rb

```diff
@@ -21,7 +23,11 @@ def initialize(params = {})
       send("#{key}=", val) if respond_to?("#{key}=")
     end
     if source == :person
-      @scope = Person.joins(:family)
+      if params[:group_category]
+        @scope = Person.eager_load(:family, :groups) # for left outer join for groups
```

Comment:
I think that should be fine, because there are no people without families currently.  If there's business logic elsewhere to enforce that, I think this should be fine.  Thoughts @seven1m ?